docs/oem-provisioning-24_04_1.md

```diff
@@ -117,7 +117,44 @@ autoinstall:
 EOF
 touch meta-data
 ```
+#### Adding additional Language packs
 
+The default Ubuntu ISO only has the English language pack seeded. To have additional languages
+available in Gnome Initial Setup, you will need to install additional language packs via the
+`autoinstall.yaml`
+
+To install additional language packs, you can make use of [late commands](https://canonical-subiquity.readthedocs-hosted.com/en/latest/reference/autoinstall-reference.html#late-commands) in the `autoinstall.yaml:
+
+```yaml
+#cloud-config
+autoinstall:
+  version: 1
+  late-commands:
+    - |
+      curtin in-target --target=/target -- bash -c '
+        # Update package list
+        apt-get update
+
+        # Installing languages not in seed
+        # Add language codes of packs you with to install here 
+        for lang in "fr" "de"; do
+            for pkg in $(check-language-support --show-installed -l "$lang"); do
+                # gimp: not installed by default
+                # fcitx5: not used by ubuntu as gnome is in use
+                # mozc-utils-gui: is in universe
+                if [ "${pkg%%-*}" = gimp ] || [ "${pkg%%-*}" = fcitx5 ] || [ "$pkg" = "mozc-utils-gui" ]; then
+                    continue
+                fi
+                apt-get install -y "$pkg"
+            done
+            for pkg in $(check-language-support --show-installed -l "$lang" -p "ibus,gnome-user-docs,gvfs"); do
```

Comment:
why do we special case those? would be nicer if the reason was documented

Reply:
My mistake, it doesn't make sense in our context, the OEM team have a different script they run, not `apt-get install`. These will be picked up by the first for loop.